### ISSUE TITLE
WCAG4.1 : typeahead -Div elements cannot be a child of a span

### ISSUE THREAD

**Issue by ShaileshDeosarkar** [corejavascript/typeahead.js#225]
WCAG Guideline 4.1 Compatible: Maximize compatibility with current and future user agents, including assistive technologies.

WCAG states - In content implemented using mark-up languages, elements have complete start and end tags, **elements are nested according to their specifications,** elements do not contain duplicate attributes, and any IDs are unique, except where the specifications allow these features.

**Comment by ShaileshDeosarkar** [comment]
[PR 226](https://travis-ci.org/github/corejavascript/typeahead.js/builds/708157774)
